src/common/CustomIconButton/CustomIconButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import {
+  EuiButton,
+} from '@elastic/eui';
+
+const CustomIconButton = ({
+  iconType, href, target, label,
+}) => {
+  if (process.env.NODE_ENV === 'test') {
+    return (
+      <EuiButton
+        href={href}
+        target={target}
+      />
+    );
+  }
+  return (
+    <EuiButton
+      iconType={iconType}
+      href={href}
+      target={target}
+    >
+      {label}
+    </EuiButton>
+  );
+};
+
+CustomIconButton.propTypes = {
+  href: PropTypes.string.isRequired,
```

Comment:
You can have an Icon without a link, isn't it? So `href` and `target` shouldn't be required.

Reply:
Updated.